hardware/setup/snap_bitstream_post.tcl

```diff
@@ -16,13 +16,8 @@
 #
 #-----------------------------------------------------------
 
-set root_dir   $::env(SNAP_ROOT)/hardware
-set fpgacard   $::env(FPGACARD)
-
-if { $fpgacard == "RCXVUP" } {
-   write_cfgmem -force -format bin -size 128 -interface SPIx8 -loadbit "up 0x0 $root_dir/viv_project/framework.runs/impl_1/psl_fpga.bit" $root_dir/viv_project/framework.runs/impl_1/psl_fpga
-} else {
-   # Max. size is 64MB for N250S and ADKU3, 128MB for S121B and N250SP. Bin file size will match the device, so -size is not relevant here.
-   write_cfgmem -force -format bin -size 128 -interface BPIx16 -loadbit "up 0x0 $root_dir/viv_project/framework.runs/impl_1/psl_fpga.bit" $root_dir/viv_project/framework.runs/impl_1/psl_fpga
-}
+set root_dir        $::env(SNAP_ROOT)/hardware
+set flash_interface $::env(FLASH_INTERFACE)
+set flash_size      $::env(FLASH_SIZE)
```

Comment:
In order to avoid  the usage of environment variables we should probably create card specific versions.

Reply:
:+1: for card specific versions of snap_bitstream_post.tcl